src/Symfony/Component/ErrorHandler/ErrorHandler.php

```diff
@@ -706,7 +706,7 @@ private function sendPhpResponse(\Throwable $exception)
             $exception = FlattenException::createFromThrowable($exception);
             $statusCode = $exception->getStatusCode();
             $headers = $exception->getHeaders();
-            $response = (new HtmlErrorRenderer(true))->render($exception);
+            $response = (new HtmlErrorRenderer(0 !== $this->scopedErrors))->render($exception);
```

Comment:
This also fix the state of the debug mode, avoiding to show the full error exception page (with stacktraces) on prod

Reply:
I've used `$this->scopedErrors` value because it's set to `0` on non-debug mode:
https://github.com/symfony/symfony/blob/b00b63366526f72ca5b9644bdebd8779c0ebd562/src/Symfony/Component/HttpKernel/EventListener/DebugHandlersListener.php#L95